docs/notebooks/alternative_links_binary.ipynb

```diff
@@ -514,7 +514,7 @@
    "cell_type": "markdown",
```

Comment:
<p>ohhh this is controversial! :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>